lua/impairative/operations.lua

```diff
@@ -221,12 +248,7 @@ function ImpairativeOperations:command_pair(args)
             if 0 < vim.v.count then
                 cmd = vim.v.count .. cmd
             end
-            local ok, result = pcall(vim.cmd, cmd)
-            if not ok then
-                result = tostring(result)
-                result = result:match"E%d*:.+" or result
-                vim.api.nvim_err_writeln(result)
-            end
+            vim.cmd(cmd)
```

Comment:
Why did you make this change? It seems unrelated to the better-n integration.

Reply:
Leftovers from debugging